DQM/HGCal/plugins/HGCalTriggerClient.cc

```diff
@@ -45,8 +45,16 @@ class HGCalTriggerClient : public DQMEDAnalyzer {
    */
   void analyze(const edm::Event&, const edm::EventSetup&) override;
 
-  MonitorElement* h_trigtime;
-  // MonitorElement *p_econdquality, *p_econdcbquality, *p_econdpayload;
+  MonitorElement *h_trigtime, *h_trigtype, *h_ECONTRawDataErrors;
+
+  // ------------ member function ------------
+  bool is0xFECAFE(const uint32_t word);
```

Comment:
I think that all mentions to `0xFECAFE` in names should be replaced with `markerWord` or some meaningful such. The actual value of the constant is not relevant to name functions/methods.

Reply:
Yes we can make those changes. What do you think about the word "blockSeparator" as a replacement for 0xfecafe ? As it separates different raw data blocks, e.g. unpacker input, output, scintillator....

Reply:
Necessary modifications have been applied (see [8165d07](https://github.com/CMS-HGCAL/cmssw/pull/86/commits/8165d077af54e5ab2fcceb6c538b92669bbe113d))